File: app_manager/appmanager/src/main/java/app/editors/manager/mvp/models/explorer/PathParts.java
package app.editors.manager.mvp.models.explorer;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class PathParts {

    @SerializedName("key")
    @Expose
    private String key = "";

    @SerializedName("path")
    @Expose
    private String path = "";

    public String getKey() {
        return key;
    }

    public void setKey(String key) {
        this.key = key;
    }

    public String getPath() {
        return path;
    }

    public void setPath(String path) {
        this.path = path;
    }

}

File: app_manager/core/src/main/java/app/documents/core/network/manager/models/explorer/PathParts.kt
package app.documents.core.network.manager.models.explorer

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class PathParts {

    @SerializedName("id")
    @Expose
    var id = ""

    @SerializedName("title")
    @Expose
    var title = ""

}